recipes-bsp/atf/atf-juno_git.bb

```diff
@@ -0,0 +1,22 @@
+require atf-96boards.inc
+
+DESCRIPTION = "ARM Trusted Firmware Juno"
+LICENSE="BSD"
+LIC_FILES_CHKSUM = "file://license.rst;md5=33065335ea03d977d0569f270b39603e"
+
+COMPATIBLE_MACHINE = "juno"
+
+DEPENDS += "u-boot-juno"
+
+SRCREV = "b762fc7481c66b64eb98b6ff694d569e66253973"
+SRC_URI = "git://github.com/ARM-software/arm-trusted-firmware.git;protocol=https;name=atf;branch=master \
+           http://releases.linaro.org/members/arm/platforms/17.04/juno-latest-oe-uboot.zip;name=junofip;subdir=juno-oe-uboot \
+"
+S = "${WORKDIR}/git"
+SRC_URI[junofip.md5sum] = "12fc772de457930fc60e42bdde97eb0a"
+SRC_URI[junofip.sha256sum] = "be1a3f8b72a0dd98ba1bf9f4fd5415d3adca052c60b090c5dccc178588ec43bc"
+
+do_compile() {
```

Comment:
a comment in the recipe will be useful to explain why we use a pre-built BL2 instead of building it ourself.